features/step_definitions/networking.rb

```diff
@@ -661,3 +661,16 @@
 
   raise "The status of condition #{type} is incorrect." unless expected_status == real_status
 end
+
+
+Given /^I run command on the#{OPT_QUOTED} node's sdn pod:$/ do |node_name, table|
+  ensure_admin_tagged
+  network_cmd = table.raw
+  node_name = node.name
```

Comment:
ops, missed this last time. 
```
node_name ||= node.name
```

Otherwise the node name specified in step will not be used but we always use current node.